admin/nyx/patches/nyx-2.1.0-devel.patch

```diff
@@ -0,0 +1,1526 @@
+diff --git a/README.md b/README.md
```

Comment:
Ehmm... this patch looks really weird to me!

You should replace it by this one: 
https://github.com/torproject/nyx/commit/dcaddf2ab7f9d2ef8649f98bb6870995ebe0b893.patch

Reply:
It is a `git diff 2.1.0..HEAD`. I'll replace it.